flux.yaml

```diff
@@ -0,0 +1,33 @@
+package:
+  name: flux
+  version: 0.38.2
+  epoch: 0
+  description: Open and extensible continuous delivery solution for Kubernetes. Powered by GitOps Toolkit.
+  target-architecture:
+    - all
+  copyright:
+    - license: Apache-2.0
+      paths:
+        - "*"
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - busybox
+      - build-base
+      - bash
+      - curl
+      - go
+      - kustomize
```

Comment:
I was wondering if kustomize is needed at runtime rather than build time?  

Adding it here will mean it's only used in the build environment when compiling but figured if a go app needs a library at compile time it would likely use go modules or something.  So am wondering if there's a runtime dependency on kustomize for the flux CLI?  If that's the case we'd need to move this up to the package section.  

i.e.
```
package:
  name: flux
  version: 0.38.2
  epoch: 0
  description: Open and extensible continuous delivery solution for Kubernetes. Powered by GitOps Toolkit.
  target-architecture:
    - all
  copyright:
    - license: Apache-2.0
      paths:
        - "*"
   dependencies:
     runtime:
      - kustomize
```

If it's used at build / compile time, ignore me :)

Reply:
We embed the Flux CRDs and controllers manifests inside the binary so people can run the install behind the firewall. These manifests are pulled from each Flux controller repo and we use kustomize remote bases to do that. The kustomize binary is only needed at build time, Flux the CLI has no runtime deps as it doesn't shell exec to anything.

Reply:
awesome - makes sense, thanks for clearing that up